mdpow/fep.py

```diff
@@ -834,8 +844,8 @@ def compress_dgdl_xvg(self):
                     # speed is similar to 'bzip2 -9 FILE' (using a 1 Mio buffer)
                     # (Since GW 0.8, openany() does not take kwargs anymore so the write buffer cannot be
                     # set anymore (buffering=1048576) so the performance might be lower in MDPOW >= 0.7.0)
-                    with open(xvg, 'r', buffering=1048576) as source:
-                        with openany(fnbz2, 'w') as target:
+                    with open(xvg, 'rb', buffering=1048576) as source:
```

Comment:
Reading in byte mode for for compatibility.

Reply:
very good